api-server/src/server/boot/challenge.js

```diff
@@ -65,10 +75,15 @@ export default async function bootChallenge(app, done) {
     backendChallengeCompleted
   );
 
+  const generateExam = createGenerateExam(app);
+
+  api.get('/generate-exam', send200toNonUser, generateExam);
```

Comment:
```suggestion
  api.get('/exam/:id', send200toNonUser, generateExam);
```
Generally speaking REST apis use path params, not query params, to identify a specific instance of a resource.  Also, we're really just 'get'ting a resource (the consumer doesn't need to care that it's generated), so we can just name the endpoint 'exam', since that's what we're getting.